### PR TITLE
Fix gitignore test

### PR DESCRIPTION
**Pull request by jayvdb**
# Pull Request Check List

Resolves: 

- [x] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

### CODE REVIEW

File: tests/test_config_gitignore.py [hunk 1]
```diff
@@ -27,6 +27,9 @@ def test_cli(runner: CliRunner) -> None:
     with open("tests/config_gitignore/.gitignore", "w") as git:
         git.write("html_two.html")
 
+    with open("tests/config_gitignore/pyproject.toml", "w") as f:
```
[the remark this answers is not in the document]
Comment (christopherpickering) [reply tests/test_config_gitignore.py:30]: yeah this is odd. Can you test something else out too - 
if you create a new folder with an html file, will it link w/out a .git, and pyproject.toml, or do you git an error?

djlint should run without a pyproject.toml file. I wonder if something is messed up in the changes I just did to the function that finds the pyproject..

[the remark this answers is not in the document]
Comment (christopherpickering) [reply tests/test_config_gitignore.py:30]: I'm going to pull this line for now as it should not be needed, we need to revisit why the test failed without it. If you have any trace do you mind sticking in discord or in a new issue?

[the remark this answers is not in the document]
Comment (jayvdb) [reply tests/test_config_gitignore.py]: Yea, ok, I'll investigate it a bit more, and create an issue tonight.

[the remark this answers is not in the document]
Comment (christopherpickering) [reply tests/test_config_gitignore.py]: thanks!

[the remark this answers is not in the document]
Comment (jayvdb) [reply tests/test_config_gitignore.py]: The tests are now passing for me on py37 & py39.  Seems it was fixed.

[the remark this answers is not in the document]
Comment (christopherpickering) [reply tests/test_config_gitignore.py]: ok nice!

**Pull request merged**